yapapi/script/__init__.py

```diff
@@ -2,6 +2,7 @@
 import itertools
 from typing import Any, Awaitable, Callable, Dict, Iterator, Optional, List, TYPE_CHECKING
 
+import yapapi
```

Comment:
Duh, this is ugly. 

I more-or-less understand why we have this, but I think the generated documentation is exactly the same before this change, so this only silences warnings. I don't think it's worth it?
Or maybe if `Script` had `:params:` section this would be different?

Reply:
no, it's not exactly the same -> without it, the type hints don't link and there's an ugly `ForwardRef` link instead...

Reply:
anyway, we definitely _do_ have a differing definition of "ugly" ... to me, it's "inelegant" at most ...

Reply:
1. I tried this once and compared result files with `diff` and got no difference. I'll try this again
2. OK, inelegant is fine (although I'd say everything related to python annotations deserves this name). Either way, I'd be glad to get rid of it if we can : )